src/java/picard/sam/AddOrReplaceReadGroups.java

```diff
@@ -30,8 +30,37 @@
  * @author mdepristo
  */
 @CommandLineProgramProperties(
-        usage = "Replaces all read groups in the INPUT file with a single new read group and assigns " +
-                "all reads to this read group in the OUTPUT BAM",
+        usage = "Replaces all read groups in the INPUT file with a single new read group and assigns all reads to this" +
+                " read group in the OUTPUT BAM.  This can be used to combine datasets to call variants on multiple" +
+                " samples.<br /><br /> " +
+                "Accepts input bam files or Global Alliance for Genomics and Health (GA4GH) URLs.  " +
```

Comment:
I don't think this is specific to this tool -- this should probably go somewhere else.